easybuild/easyconfigs/t/Trim_Galore/Trim_Galore-0.4.4-intel-2017a.eb

```diff
@@ -0,0 +1,33 @@
+# 0.4.2 modified by:
+# Adam Huffman
+# The Francis Crick Institute
+# TODO: deal with the hardcoding of /usr/bin/perl
+
+easyblock = "PackedBinary"
+
+name = 'Trim_Galore'
+version = '0.4.4'
+
+homepage = 'http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/'
+description = """A wrapper tool around Cutadapt and FastQC to consistently apply 
+ quality and adapter trimming to FastQ files, with some extra functionality for 
+ MspI-digested RRBS-type (Reduced Representation Bisufite-Seq) libraries."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+sources = ['%(namelower)s_v%(version)s.zip']
+source_urls = ['http://www.bioinformatics.babraham.ac.uk/projects/%(namelower)s']
+
+install_cmd = "cp -a * %(installdir)s && chmod +x %(installdir)s/%(namelower)s"
+
+dependencies = [
+    ('FastQC', '0.11.5', '-Java-1.8.0_74', True),
+    ('cutadapt', '1.14', '-Python-2.7.13'),
+]
+
+sanity_check_paths = {
+    'files': ["trim_galore"],
```

Comment:
copy this to `/bin` subdir?

Reply:
I don't see the point of that?

Reply:
automagically added to `$PATH`, no need for custom `modextrapaths`?